### ISSUE TITLE
Change Body Text Color for Accessibility

### ISSUE THREAD

**Issue by ErinRosie** [chimera/sonomafireinfo.org#32]
Our light grey body copy does not meet accessibility standards.

Please change to: #6B737B

**Comment by ErinRosie** [comment]
Mike may have already done something with this. Not sure? We talked about it via Slack, but I wanted to be sure it was logged somewhere.
